drivers/dma/dma_mcux_lpc.c

```diff
@@ -484,13 +547,19 @@ static int dma_mcux_lpc_init(const struct device *dev)
 	size_channel_data =
 		sizeof(struct call_back) * config->num_of_channels;
 	data->data_cb = k_malloc(size_channel_data);
+	data->linked_otrig_ch = k_malloc(sizeof(uint8_t) * config->num_of_otrigs);
```

Comment:
please don't use k_malloc, I'm doing another PR where I'm changing this driver to pre kernel and removing all the k_mallocs from the init

Reply:
Malloc is no longer used